### ISSUE TITLE
Logging  in `eyra` seems to be broken with new `Origin` release

### ISSUE THREAD

**Issue by benny-n** [sunfishcode/origin#63]
Hi, 

Cloning `eyra` and trying to run the hello-world crate example with `log` and `env_logger` features enabled seem to yield the next  compilation error: 
```
   Compiling origin v0.13.7
error[E0433]: failed to resolve: use of undeclared crate or module `thread`
  --> /home/benny/.cargo/registry/src/index.crates.io-6f17d22bba15001f/origin-0.13.7/src/log.rs:23:13
   |
23 |             thread::current_thread_id().as_raw_nonzero()
   |             ^^^^^^ use of undeclared crate or module `thread`
   |
help: consider importing this module
   |
1  + use crate::thread;
   |

For more information about this error, try `rustc --explain E0433`.
error: could not compile `origin` (lib) due to previous error
```

The exact steps I performed:
```
git clone https://github.com/sunfishcode/eyra
cd eyra/example-crates/hello-world
cargo add eyra -F log,env_logger
cargo run
```
Other info:
OS: Arch Linux x86_64

I experimented with `eyra` yesterday (29/09/23) and didn't see this issue, could this be caused by the new `Origin` release? (specifically [this commit](https://github.com/sunfishcode/origin/commit/21acb74cc822854274631c477731ef955cd1a028) since the other ones are just docs or formatting)

**Comment by sunfishcode** [comment]
Thanks for the report! I've now posted https://github.com/sunfishcode/origin/pull/64 which includes a fix.

**Comment by sunfishcode** [comment]
This is now fixed in origin 0.13.8.
